GiniVision.podspec

```diff
@@ -15,11 +15,24 @@ The Gini Vision Library for iOS provides functionality to capture documents with
   s.author           = { 'Peter Pult' => 'p.pult@gini.net' }
   s.source           = { :git => 'https://github.com/gini/gini-vision-lib-ios.git', :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/gini'
-
   s.ios.deployment_target = '8.0'
+  s.default_subspec = 'Core'
+
+  s.subspec 'Core' do |core|
+    core.source_files = 'GiniVision/Classes/Core/**/*'
+    core.resources = 'GiniVision/Assets/*'
+  end
+
+  s.subspec 'Networking' do |networking|
+    networking.source_files = 'GiniVision/Classes/Networking/*.swift', 'GiniVision/Classes/Networking/Extensions/*.swift'
+    networking.dependency "Gini-iOS-SDK", "~> 0.6.0"
+  end
 
-  s.source_files = 'GiniVision/Classes/**/*'
-  s.resources = 'GiniVision/Assets/*'
+  s.subspec 'Networking+Pinning' do |pinning|
+    pinning.source_files = 'GiniVision/Classes/Networking/Pinning/*', 'GiniVision/Classes/Networking/*.swift', 'GiniVision/Classes/Networking/Extensions/*'
```

Comment:
Why are you adding the networking and networking extensions - they should be already added by the `Gini-iOS-SDK/Pinning` dependency?

Reply:
They are swift `Extensions` for the Networking, they are not part of the pinning.